docs/list.json

```diff
@@ -1,1048 +1,936 @@
 {
-
-	"en": {
```

Comment:
The formatting of the existing sections should not be changed.

Reply:
This doesn't look correct yet. The diff is way to large...

Reply:
I'm sorry. 
I thought it was fixed and pushed it, but I guess it wasn't.
I'll check locally again.